source/configure/configuration-settings.rst

```diff
@@ -51,9 +51,9 @@ The name of the environment variable for any setting can be derived from the nam
 
 1. Find the setting in ``config.json``. In this case, *ServiceSettings.SiteURL*.
 2. Add ``MM_`` to the beginning and convert all characters to uppercase and replace the ``.`` with ``_``. For example, *MM_SERVICESETTINGS_SITEURL*.
-3. The setting becomes ``export MM_SERVICESETTINGS_SITEURL="http://example.com"``.
+3. The setting becomes ``export MM_SERVICESETTINGS_SITEURL="http://example.com"``. An environment variable (ENV) file is created under `/etc/mattermost`.
```

Comment:
> An environment variable (ENV) file is created under `/etc/mattermost`.

Can you clarify this line? I think the admin needs to manually create this file and this is only needed when Mattermost is deployed via an init file, and the entry also needs to be added in the init file. Basically, it can be any file that the user wishes.

Did you mean to say all of that? Or something else?

Reply:
I could use your help clarifying this information, @agnivade. This docs request is coming from a [Community-based conversation](https://community-daily.mattermost.com/core/pl/hdme69zqwjgh5qaurc4oqsbexw). Would you be open to drafting an update?

Reply:
I took a look at the thread. It seems the user is not the person who configured the system.

Basically there are 2 things here:
1. A config setting configured via an env var cannot be modified from the system console. This is already documented.
2. If Mattermost is run from an init file, the environment variables can either be set via `Environment=<>`, or `EnvironmentFile=<path/to/file>`. In the second case, the file should contain the list of environment variables to set. More info here: https://www.man7.org/linux/man-pages/man5/systemd.exec.5.html#ENVIRONMENT

I think 1 is already documented. I'll leave it to you to see how/where you want to incorporate 2.

Reply:
Thank you so much for your help with this, @agnivade.